tests/Kron.py

```diff
@@ -389,6 +486,71 @@ def getTolFlux(self, a, b, kfac):
 
         return tol
 
+    def testForced(self):
+        """Check that forced photometry works in the presence of rotations and translations"""
+        kfac = 2.5
+        warper = afwMath.Warper("lanczos4")
+        a = 13
+        for axisRatio in (0.25, 1.0):
+            b = a*axisRatio
+            for theta in (0, 30, 45):
+                width, height = 256, 256
+                center = afwGeom.Point2D(0.5*width, 0.5*height)
+                original = makeGalaxy(width, height, 1000.0, a, b, theta)
+                msConfig = makeSourceMeasurementConfig(kfac=kfac)
+                source = measureFree(original, center, msConfig)
+                if source.get("extensions_photometryKron_KronFlux_flag"):
+                    continue
+                angleList = [45, 90,]
+                scaleList = [1.0, 0.5]
+                offsetList = [(1.23, 4.56), (12.3, 45.6)]
+
+                for angle in angleList:
+                 for scale in scaleList:
+                  for offset in offsetList:
+#angle, scale, offset in itertools.product(angleList, scaleList, offsetList):
```

Comment:
I was sloppy to have left this in the unit test.  But the reason for it was
that the product notation didn't work with the python I was using, and I
wasn't sure why.  I split it up as a temporary measure, but forgot to come
back to it later and make a decision about how the code should really be.

On Tue, Feb 17, 2015 at 11:36 AM, Paul Price <notifications@github.com>
wrote:

> In tests/Kron.py
> <https://github.com/lsst/meas_extensions_photometryKron/pull/1#discussion_r24846090>
> :
>
> > +            for theta in (0, 30, 45):
> > +                width, height = 256, 256
> > +                center = afwGeom.Point2D(0.5*width, 0.5*height)
> > +                original = makeGalaxy(width, height, 1000.0, a, b, theta)
> > +                msConfig = makeSourceMeasurementConfig(kfac=kfac)
> > +                source = measureFree(original, center, msConfig)
> > +                if source.get("extensions_photometryKron_KronFlux_flag"):
> > +                    continue
> > +                angleList = [45, 90,]
> > +                scaleList = [1.0, 0.5]
> > +                offsetList = [(1.23, 4.56), (12.3, 45.6)]
> > +
> > +                for angle in angleList:
> > +                 for scale in scaleList:
> > +                  for offset in offsetList:
> > +#angle, scale, offset in itertools.product(angleList, scaleList, offsetList):
>
> The product has been split into separate for loops. Note that the
> indentation is wrong, and splitting it up like this is unnecessary because
> the individual elements aren't used separately.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lsst/meas_extensions_photometryKron/pull/1/files#r24846090>
> .
>